### DISCUSSION TITLE
How to auto-label my request? (Answered)

### DISCUSSION THREAD

**Discussion by claudejdev** [PygmalionOfCyprus/cmo-db-requests#3943] [Q&A] [answered]
_PygmalionOfCyprus said:_
> Please precede the "Affected DBID(s)"/"DBID to Copy" with an annex name going forward (e.g., "Weapon 718") so the auto-labeler can parse it. 🙂
Ref.: https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/3068#issuecomment-1460704230, https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/3160#issuecomment-1462524043, https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/3042#issuecomment-1460695147, https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/3046#issuecomment-1460698640

Paraphrasing @PygmalionOfCyprus here as I feel it's worth mentioning that neat little feature in the Q&A to let the world know.

**Comment by Parel803** [comment]
thanks for sharing it here.
regards GJ

**Comment by PygmalionOfCyprus** [comment, marked as answer]
I've already updated the Update and Copy Entry forms to mention this. Whether people read it is a different question...
